step.sh

```diff
@@ -1,22 +1,57 @@
 #!/bin/bash
-#
-# Upload to Autify
-set -e
+set -ex
 
-# shellcheck disable=SC2154
-export AUTIFY_UPLOAD_TOKEN="$upload_token"
-# shellcheck disable=SC2154
-export AUTIFY_PROJECT_ID="$project_id"
-# shellcheck disable=SC2154
-export AUTIFY_APP_DIR_PATH="$app_dir_path"
+ARGS=()
 
-export AUTIFY_MOBILE_SCRIPT="https://raw.githubusercontent.com/autifyhq/autify-for-mobile-cli/main/autify_mobile_cli.sh"
-export AUTIFY_MOBILE_SCRIPT_NAME="autify_mobile_cli.sh"
+function add_args() {
+  ARGS+=("$1")
+}
 
-readonly WORKIND_DIR="./"
+function exit_script() {
+  local code=$?
+  envman add --key AUTIFY_UPLOAD_EXIT_CODE --value "${code}"
+}
+trap exit_script EXIT
 
-# download script
-curl -L -o "$WORKIND_DIR/$AUTIFY_MOBILE_SCRIPT_NAME" $AUTIFY_MOBILE_SCRIPT
-chmod 777 "$WORKIND_DIR/$AUTIFY_MOBILE_SCRIPT_NAME"
+# Install Autify CLI
+curl https://autify-cli-assets.s3.amazonaws.com/autify-cli/channels/stable/install-cicd.bash | bash -xe
```

Comment:
**It is not a prerequisite** because mobile-cli has been installed on-demand from GitHub in the past. According to https://github.com/bitrise-io/bitrise-steplib/pull/3317#discussion_r768511098, Bitrise does not consider it a blocker. 

However, Bitrise recommends resolving dependencies as follows, so please check once if you has not.

https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md#do-not-use-submodules-or-require-any-other-resource-downloaded-on-demand

Reply:
See https://github.com/bitrise-io/bitrise-steplib/pull/3594#issuecomment-1232158044

I can't find any way to install external dependency via `apt`. I've already raised this to Bitrise team.

Reply:
There are several steps doing similar e.g. https://github.com/pilvikala/bitrise-step-snyk/blob/main/step.sh#L9

Reply:
Okey. Thank you for checking